File: Homework3/Question2/src/HoodedPitohui.java
public class HoodedPitohui extends Bird {
    public HoodedPitohui() {
        super("orange-black", 4000);
    }

    @Override
    public boolean isEdible() {
        return false;
    }

    public static void testHoodedPitohui() {
        HoodedPitohui hoodedPitohui = new HoodedPitohui();
        //Test hoodedPitohui
        System.out.println(hoodedPitohui.isEdible() == false);
        System.out.println(hoodedPitohui.getColor() == "orange-black");
        System.out.println(hoodedPitohui.getNumberOfFeathers() == 4000);
    }
}

File: Homework3/HoodedPitohui.kt
class HoodedPitohui : Bird {
    constructor() : super("orange-black", 4000)

    override fun isEdible(): Boolean {
        return false
    }

    companion object {
        fun testHoodedPitohui() {
            val hoodedPitohui = HoodedPitohui()
            println(hoodedPitohui.isEdible() == false)
            println(hoodedPitohui.getColor() == "orange-black")
            println(hoodedPitohui.getNumberOfFeathers() == 4000)
        }
    }
}